src/Bricks/ArticleValueObject.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Astrogoat\Blog\Bricks;
+
+use Helix\Lego\Bricks\ValueObjects\BrickValueObject;
+
+class ArticleValueObject extends BrickValueObject
+{
+    protected $cache = [];
+
+    public function __construct(protected $value)
+    {
+    }
+
+    public function getArticleModel()
+    {
+        if (isset($this->cache[$this->value])) {
+            return $this->cache[$this->value];
+        }
+
+        $this->cache[$this->value] = \Astrogoat\Blog\Models\Article::find($this->value);
+
+        return $this->cache[$this->value];
+    }
+
+    public function getValue()
+    {
+        return $this->value;
+        $selectedArticle = \Astrogoat\Blog\Models\Article::find($this->value);
+
+        return $selectedArticle != null ? $selectedArticle->title : '';
+    }
+
+    public function forJavascript()
+    {
+        return $this->getValue() ?? '';
+    }
+
+    public function __toString()
+    {
+        return '';
```

Comment:
I think something should be returned here as it might cause confusion. possibly a json string of the title/author/data? If not maybe throw an error that the function has not been implemented.